components/dropdownbutton/icons.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Icons
+page_title: DropDownButton Icons
+description: How to use icons with the DropDownButton for Blazor.
+slug: dropdownbutton-icons
+tags: telerik,blazor,dropdownbutton,icons
+published: True
+position: 5
+---
+
+# DropDownButton Icons
+
+The `<TelerikDropDownButton>` and `<DropDownButtonItem>` provide an `Icon` parameter to add an icon to the main button and all secondary action items.
+
+The `Icon` parameter type is `object` and it accepts:
+
+* a member of the [`FontIcon` enum]({%slug general-information/font-icons%}#icons-list)
+* a property of the static [`SvgIcon` class]({%slug general-information/font-icons%}#icons-list)
+* a `string` that is a CSS class for a custom icon
+
+>caption How to use icons in Telerik Blazor DropDownButton
+
+````CSHTML
+<TelerikDropDownButton Icon="@("sln")">
+    <DropDownButtonContent>Telerik Icon</DropDownButtonContent>
+    <DropDownButtonItems>
+        <DropDownButtonItem Icon="@FontIcon.User">Telerik Font Icon</DropDownButtonItem>
+        <DropDownButtonItem Icon="@SvgIcon.Gear">Telerik SVG Icon</DropDownButtonItem>
+        <DropDownButtonItem Icon="@CustomIconClass">Custom Icon</DropDownButtonItem>
+        <DropDownButtonItem> <TelerikLoader /> Custom markup </DropDownButtonItem>
+    </DropDownButtonItems>
+</TelerikDropDownButton>
+
+<style>
+    /* Third-party icon libraries should provide these styles out-of-the-box. */
+
+    .my-icon {
+        width: 1em;
+        height: 1em;
+        font-size: 16px;
+        background: purple;
+    }
+</style>
+
+@code {
+    string CustomIconClass { get; set; } = "my-icon";
+}
+````
+
+>tip You can also add custom icons and images with additional markup inside the `<DropDownButtonContent>`, where the text is.
+
+>note Images used as icons should generally be small enough to fit in a line of text. The primary button is an inline element and is not designed for large images. If you want to use big icon buttons, consider one of the following options:
```

Comment:
This is a huge note that looks funny once you build the article. Consider turning it into a new document section by adding an H2, for example, "## Best Practices".

![image](https://github.com/telerik/blazor-docs/assets/60105689/3435df3a-f86e-4401-9220-7c32faee2bc4)